popup/translations.ts

```diff
@@ -40,15 +49,20 @@ export const getTranslation = (translationKey, ...args) => {
 
 /* eslint-disable no-param-reassign */
 export const createTextSetter = (
-  element,
-  translationKey,
+  element: Element,
+  translationKey: string,
   property = 'innerText'
 ) => {
   if (setterHooks.some((hook) => hook.element === element)) {
     throw new Error('This element already has a text setter')
   }
 
-  const hook = {
+  const hook: {
+    element: Element
+    translationKey: string
+    property: string
```

Comment:
The type here should be `property: keyof HTMLElement` or `property: keyof Element`